ui/src/routes/+page.svelte

```diff
@@ -1,2 +1,150 @@
+            station: StationType.Station3,
+            connection: true,
+            dsLinkActive: false,
+            radioLink: false,
+            rioLink: false,
+            linkActive: false,
+            isEnabled: false,
+            isAuto: false,
+            isBypassed: false,
+            isEStopPressed: false,
+            isEStopped: false,
+            battery: 0,
+            monitorStatus: MonitorStatusType.Unknown,
+            averageTripTime: 0,
+            lostPackets: 0,
+            signal: 0,
+            dataRateTotal: 0,
+            stationStatus: DSStationStatus.Waiting
+        },
+        {
+            teamNumber: 2468,
+            alliance: AllianceType.Red,
+            station: StationType.Station1,
+            connection: true,
+            dsLinkActive: true,
+            radioLink: false,
+            rioLink: false,
+            linkActive: false,
+            isEnabled: false,
+            isAuto: false,
+            isBypassed: false,
+            isEStopPressed: false,
+            isEStopped: false,
+            battery: 0,
+            monitorStatus: MonitorStatusType.Unknown,
+            averageTripTime: 0,
+            lostPackets: 0,
+            signal: 0,
+            dataRateTotal: 0,
+            stationStatus: DSStationStatus.Unknown
+        },
+        {
+            teamNumber: 1357,
+            alliance: AllianceType.Red,
+            station: StationType.Station2,
+            connection: true,
+            dsLinkActive: true,
+            radioLink: true,
+            rioLink: true,
+            linkActive: false,
+            isEnabled: false,
+            isAuto: false,
+            isBypassed: false,
+            isEStopPressed: false,
+            isEStopped: false,
+            battery: 0,
+            monitorStatus: MonitorStatusType.Unknown,
+            averageTripTime: 0,
+            lostPackets: 0,
+            signal: 0,
+            dataRateTotal: 0,
+            stationStatus: DSStationStatus.Unknown
+        },
+        {
+            teamNumber: 3242,
+            alliance: AllianceType.Red,
+            station: StationType.Station3,
+            connection: true,
+            dsLinkActive: true,
+            radioLink: true,
+            rioLink: true,
+            linkActive: true,
+            isEnabled: false,
+            isAuto: false,
+            isBypassed: false,
+            isEStopPressed: false,
+            isEStopped: false,
+            battery: 0,
+            monitorStatus: MonitorStatusType.Unknown,
+            averageTripTime: 0,
+            lostPackets: 0,
+            signal: 0,
+            dataRateTotal: 0,
+            stationStatus: DSStationStatus.Unknown
+        }
+    ];
+
+    let detailView = () => {};
+</script>
+
+<div class="grid grid-cols-fieldmonitor lg:grid-cols-fieldmonitor-large gap-0.5 md:gap-1 mx-auto justify-center">
+    <MonitorRow monitorFrame={teams[0]} station="blue1" detailView={detailView} />
```

Comment:
Would prefer if we used our AllianceType and StationType enums here, rather than having different representations of alliance and station throughout the entire app.